src/store/modules/credential-type.ts

```diff
@@ -18,15 +18,16 @@ const state: State = {
 };
 
 const getters = {
-  pagedCredentialTypes: (state: State): IApiPagedResult<ICredentialType> =>
-    state.page,
+  pagedCredentialTypes: (state: State): any =>
+    state.page.results.map(item=>{return {text:item.description, value:item.id}})
```

Comment:
Don't modify the output in existing getters, as it may interfere with any other components using this getter. Either create another getter for your purpose or process the data in your component according to the output it requires, especially if it is the only place where the output needs to be formatted a specific way.